src/app/payment/paymentMethod/PaymentMethodTitle.tsx

```diff
@@ -119,7 +119,7 @@ function getPaymentMethodTitle(
             },
             [PaymentMethodId.Mollie]: {
                 logoUrl: method.method === 'credit_card' ? '' : cdnPath(`/img/payment-providers/${method.method}.svg`),
-                titleText: methodName,
+                titleText: method.method === 'credit_card' ? 'Credit Card' : methodName,
```

Comment:
should this text be translatable?

Reply:
@capsula4  what do you think about using lodash `startCase`! to convert 'Credit card' to 'Credit Card'